config/locales/en.yml

```diff
@@ -199,9 +199,9 @@ en:
     reset_password_heading: Set your new password
     password_changed: Password changed successfully
     reset_password_legend: You can reset your password by entering the code you have received via email.
+    old_password_lbl: Old password
```

Comment:
why is this being changed?

Reply:
<img width="767" alt="Screen Shot 2019-12-10 at 16 55 42" src="https://user-images.githubusercontent.com/41922771/70550643-f724f100-1b6d-11ea-881b-093daea399f3.png">

I think I changed it to be more consistent with the error messages associated with password. But I'm not sure if that is a strong enough rationale so I have changed it back

Reply:
The content has been signed off by content people and everything should be as per the protype, so let's not make any content changes...